doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -73,6 +73,7 @@
     - `Pimcore\Bundle\AdminBundle\Security\User\UserLoader` => `Pimcore\Security\User\UserLoader`
     - `Pimcore\Bundle\AdminBundle\Security\User\TokenStorageUserResolver` => `Pimcore\Security\User\TokenStorageUserResolver`
   - `pimcore_admin.serializer` service has been deprecated and will be removed in Pimcore 11. Please use `pimcore.serializer` instead.
+- [DataObjects] - Deprecated Button control for DataObjects layout definition. It will be removed in Pimcore 11. You can define your custom layout component to replace this element like described [here](../../20_Extending_Pimcore/13_Bundle_Developers_Guide/12_Adding_Object_%20Layout_types.md)
```

Comment:
```suggestion
- [DataObjects] - Deprecated Button control for DataObjects layout definition. It will be removed in Pimcore 11. You can define your own custom layout component to replace this element like described [here](../../20_Extending_Pimcore/13_Bundle_Developers_Guide/12_Adding_Object_%20Layout_types.md)
```